src/Common/src/Interop/User32/Interop.CloseDesktop.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        [DllImport(Libraries.User32, ExactSpelling = true)]
```

Comment:
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-closedesktop#return-value
```suggestion
        [DllImport(Libraries.User32, ExactSpelling = true,  SetLastError = true)]
```